pkg/utils/functions.go

```diff
@@ -113,6 +115,39 @@ func IsThirdBitmap(bitmap string) bool {
 	return false
 }
 
+// Get message format
+func MessageFormat(buf []byte) string {
```

Comment:
What's the goal of this function? I don't see we use it in the code only in tests.

Reply:
The function will use command and web server in next PR

Reply:
Is it possible to move it from this branch into a new one?

Reply:
removed